src/domain/alerts/contracts/safe-decoder.helper.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { Hex, encodeFunctionData, getAddress } from 'viem';
+import { faker } from '@faker-js/faker';
+import { SafeDecoder } from '@/domain/alerts/contracts/safe-decoder.helper';
+
+describe('SafeDecoder', () => {
+  let target: SafeDecoder;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    target = new SafeDecoder();
+  });
+
+  it('decodes an addOwnerWithThreshold function call correctly', () => {
+    const owner = faker.finance.ethereumAddress() as Hex;
+    const threshold = faker.number.bigInt();
+
+    const data = encodeFunctionData({
+      abi: target.abi,
+      functionName: 'addOwnerWithThreshold',
+      args: [owner, threshold],
+    });
+
+    expect(target.decodeFunctionData({ data })).toEqual({
+      functionName: 'addOwnerWithThreshold',
+      args: [getAddress(owner), threshold],
+    });
+  });
+
+  it('logs if the function call cannot be decoded', () => {
```

Comment:
Should it be "trows if" instead of "logs if"?

Reply:
Indeed. I've fixed it in 0d89ce993ba48efe4bc94ea60ce9bca006f5c226.